### DISCUSSION TITLE
[Suggestion] Introduce --skip-partition option (PostgreSQL)

### DISCUSSION THREAD

**Discussion by h6ah4i** [sqldef/sqldef#638] [Feature requests]
When there are a large number of partitioned tables in the database (e.g., 1,000+), it takes a significant amount of time to dump the tables. Currently, I'm using the `skip_tables` option in the YAML configuration file to ignore comparison of partitioned tables, but since this option does not affect the table dumping process, the operation speed does not improve.

Moreover, partitioned tables are not currently supported by the `psqldef` command and cannot be handled properly. Therefore, I would like to temporarily resolve this issue by introducing a `--skip-partition` option, similar to `--skip-view` and `--skip-extension`.

What do you think about this proposal?

**Comment by chumaltd** [comment]
@h6ah4i Could you try latest v0.17.29?  
It should exclude all partitions - having data - at the first step.

I'm also interested about actual behavior with partitioning.
Thanks.

**Comment by k0kubun** [comment]
Thanks @chumaltd for that PR. I'm open to merging a PR that implements `--skip-partition` too.

**Comment by gfx** [comment]
Added `--skip-partition` to psqldef in https://github.com/sqldef/sqldef/pull/1036
